src/main.js

```diff
@@ -0,0 +1,33 @@
+const Discord = require('discord.js');
+
+const client = new Discord.Client();
+
+const prefix = "?EXAMPLE"
+
+client.once('ready', () => {
+	console.log('hello');
+});
+
+/*fn: (msg) => {
+	let application = {}
+	let filter = (msg) => !msg.author.bot
+	let options = {
+		max: 1,
+		time: 15000
+	};
+}*/
+
+client.on('message', message => {
+	if(!message.content.startsWith(prefix) || message.author.bot) return;
+
+	const args = message.content.slice(prefix.length).split(/ +/);
+	const command = args.shift().toLowerCase();
+
+	console.log(args);
+	message.channel.send(args);
+})
```

Comment:
could you move this to `index.ts`? most of the setup work you did here was taken care of there, and then we'll be good to go!